src/perfcollect/perfcollect

```diff
@@ -642,6 +642,10 @@ usePerf=1
 # Use LTTng
 useLTTng=1
 
+# Use EventPipe to collect CLR events
+useEventPipe=0
```

Comment:
I'd like to propose that instead of using the term `eventpipe` that we use `dotnetTrace` and/or `dotnet-trace` since the actual tool being used here is `dotnet-trace`.  This is more of a forward-looking thing, so that it's clear what collector is being used should there be multiple, or if people don't know what `eventpipe` is.

Reply:
will do